chrome/content/zotero/zoteroPane.js

```diff
@@ -1893,6 +1909,21 @@ var ZoteroPane = new function()
 		}
 		this.selectItem(item.id).done();
 	}
+
+
+	/**
+	 * Return whether every selected item is deleted.
```

Comment:
Instead of having "Return" here (which would get real repetitive and is redundant with `@return`) and explaining the actual test (which could change), this can just be a more general description of what the function does. E.g., "Check whether all selected items can be restored from trash".

Also, nit: No need for periods at the end of comments

And same for the other function